great-idea-website/index.html

```diff
@@ -15,49 +15,70 @@
 </head>
 
   <body>
+  <nav class="navigation">
+      <a href="#">Services</a>
+      <a href="#">Product</a>
+      <a href="#">Vision</a>
+      <a href="#">Features</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+  </nav>
+  
+<header class="header">
+  <img class="idea-img" src="img/logo.png" alt="Great Idea! Company logo.">
+  <h1 class="innovation-header">Innovation</br>On</br>Demand</h1>
 
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
+  <h2 class="getstarted-button">Get Started</h2>
```

Comment:
You could name this button's class "call-to-action" or "cta" - this is a term for buttons or text on websites that direct a user to do something like 'get started' :)